src/main/webapp/WEB-INF/views/UserPage.html

```diff
@@ -0,0 +1,388 @@
+                                            <option value="2">Venture capital</option>
+                                            <option value="3">Mergers and acquisitions</option>
+                                        </select>
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label class="col-md-4 control-label" for="addInterest"></label>
+                                    <div class="col-md-4 btn-group">
+                                        <button id="addInterest" class="btn btn-success" type="submit">Add Interest
+                                        </button>
+                                        <button class="btn btn-warning" type="reset">Clean</button>
+                                    </div>
+                                </div>
+
+                            </fieldset>
+                        </form>
+                    </div>
+                    <div class="container">
+                        <form class="form-horizontal">
+                            <fieldset>
+                                <legend>Add new event</legend>
+                                <div class="form-group">
+                                    <label class="col-md-4 control-label" for="textinput">Name</label>
+                                    <div class="col-md-4">
+                                        <input type="text" placeholder="Event name"
+                                               class="form-control input-md">
+                                    </div>
+                                </div>
+                                <div class="form-group">
+                                    <label class="col-md-4 control-label" for="addProject"></label>
+                                    <div class="col-md-4">
+                                        <div class="btn-group">
+                                            <button name="addEvent" class="btn btn-success"
+                                                    type="submit">Add Event
+                                            </button>
+                                            <button name="addProject" class="btn btn-warning" type="reset">Clean
+                                            </button>
+                                        </div>
+
+                                    </div>
+                                </div>
+                            </fieldset>
+                        </form>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<!--------------------------------------------------------------------------------------->
+<div class="footer">
+    <div class="container">
+        <div class="row centered">
+            <a href="#"><i class="fa fa-twitter"></i></a>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)